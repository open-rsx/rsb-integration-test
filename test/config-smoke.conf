[qualityofservice]
reliability = UNRELIABLE
ordering = UNORDERED

[errorhandling]
onhandlererror = EXIT

[transport.inprocess]
enabled = 0

[transport.spread]
host    = foo
port    = 9876
enabled = 1

[transport.spread.converter.cpp]
utf-8-string = std::string

[transport.spread.converter.python]
utf-8-string = str

[transport.spread.converter.java]
utf-8-string = String

[transport.spread.converter.lisp]
utf-8-string = string
