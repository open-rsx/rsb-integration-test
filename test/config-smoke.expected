qualityofservice.reliability: UNRELIABLE
qualityofservice.ordering: UNORDERED
errorhandling.onhandlererror: EXIT
transport.inprocess.enabled: @FALSE@
transport.spread.host: foo
transport.spread.port: 9876
transport.spread.enabled: @TRUE@
transport.spread.converter.@LANG@.string: @STRINGTYPE@
